cmd/kube-apiserver/apiserver.go

```diff
@@ -32,6 +34,8 @@ import (
 
 func main() {
 	runtime.GOMAXPROCS(runtime.NumCPU())
+	rand.Seed(time.Now().UTC().UnixNano())
```

Comment:
IMO it's not super important to seed rand, but it doesn't hurt either I suppose.